test/Microsoft.AspNet.Mvc.FunctionalTests/ActivatorTests.cs

```diff
@@ -187,6 +187,7 @@ public class ActivatorTests
                            "<input id=\"foo\" name=\"foo\" type=\"hidden\" value=\"test content\" />" +
```

Comment:
we have test classes such as `DefaultControllerActivatorTest`.  probably should have some unit testing of `DefaultTagHelperActivator`.  let's have a quick conversation tomorrow about whether any unit testing additions are needed in this PR.